components/Article.tsx

```diff
@@ -55,7 +55,12 @@ const Article: React.FunctionComponent<Props> = props => {
         )
     }
     return (
-        <PageLayout contentTitle={props.alternateTitle ?? props.title} metaTags={metaTags} leftColumn={tocFragment}>
+        <PageLayout
+            title={props.alternateTitle}
```

Comment:
I agree it's a bit confusing. The `PageLayout` and `Layout` components are just concerned with the document title (`<title>` and meta tags) so the meaning of "title" here is more general than the article title. That's because we have other pages that are not articles. `contentTitle` was introduced in order to add the site title suffix behavior.

An alternative that I was thinking about is to change the props to `documentTitle` and a boolean to toggle the site title suffix behavior, instead of `title`/`contentTitle` pair.

Reply:
That sounds like a good plan to me!